cmake/AddGRBInstall.cmake

```diff
@@ -112,37 +100,40 @@ endif()
 # for each enabled backend, add its information for the wrapper generation
 # paths may have spaces, hence wrap them inside single quotes ''
 
+# all backends include both REFERENCE_INCLUDE_DEFS and REFERENCE_OMP_INCLUDE_DEFS,
+# due to the dependency on OpenMP -- to be resolved
+
 # shared memory backends
 if( WITH_REFERENCE_BACKEND )
-	addBackendWrapperGenOptions( "reference"
-		COMPILE_DEFINITIONS "${REFERENCE_SELECTION_DEFS}"
-		LINK_FLAGS "'${SHMEM_BACKEND_INSTALL_DIR}/lib${BACKEND_LIBRARY_OUTPUT_NAME}.a'"
-			"'${ALP_UTILS_INSTALL_DIR}/lib${ALP_UTILS_LIBRARY_OUTPUT_NAME}.a'" "${NUMA_LFLAG}"
```

Comment:
I think this guy is now being missed by grbcxx, causing the link error commented earlier @alberto-scolari